contributte-datagrid/app/Presenters/AbstractPresenter.php

```diff
@@ -51,4 +48,7 @@ public function changeStatus($id, string $newStatus): void
 			$this->redirect('this');
 		}
 	}
+
+
+	abstract public function createComponentGrid(): DataGrid;
```

Comment:
This should be good enough replacement for `if (!$this['grid'] instanceof DataGrid) {` check